<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Template</class>
 <widget class="QDialog" name="Template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Название шаблона</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_title"/>
   </item>
   <item>
    <widget class="QLabel" name="labelExample">
     <property name="text">
      <string>Пример: | 16:00 | 220 | video1.flv | 47jh-eda2-uzh7...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableSchedule">
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>25</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>start</string>
      </property>
      <property name="icon">
       <iconset theme="dark">
        <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>duration (min)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>filename</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>key</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
